grant-project/grant-monorepo
# Fix proposal body truncation

### What This Does

Uses the Magic of Flexbox™️ to make the proposal body the same height as the sidebar, rather than the hard-coded max height of before.

### Steps to Test

* Make the sidebar really big, make sure the proposal body adjusts.
* Make the sidebar really small, make sure the proposal body adjusts.
* Make sure when doing the above that the "Read more" button only appears if the body is truncated (May require a screen resize if you are manually editing heights in inspector.)

## Screenshots

### Regular bod

<img width="1302" alt="screen shot 2018-09-25 at 3 26 44 pm" src="https://user-images.githubusercontent.com/649992/46038361-d5f6f280-c0d8-11e8-832b-cf6aadfa36f6.png">


### Small bod

<img width="1316" alt="screen shot 2018-09-25 at 3 26 58 pm" src="https://user-images.githubusercontent.com/649992/46038341-cecfe480-c0d8-11e8-89ab-b550d1eef197.png">

### B I G B O D

<img width="674" alt="screen shot 2018-09-25 at 3 27 46 pm" src="https://user-images.githubusercontent.com/649992/46038370-dc856a00-c0d8-11e8-9580-364332272b09.png">
